tests/schemas_test.py

```diff
@@ -0,0 +1,28 @@
+from nio.schemas import check_user_id
+
+
+class TestClass:
+    def test_check_user_id__valid(self):
+        assert check_user_id("@foobar:example.com") is True
+
+    def test_check_user_id__invalid(self):
+        try:
+            check_user_id("foobar:example.com")
+        except ValueError:
+            pass
```

Comment:
Use 
```py
with pytest.raises(ValueError):
    check_user_id(...)
```
instead of try/except blocks, that will be shorter and will ensure the exception occurs to pass the test.